src/kernel/chainparams.cpp

```diff
@@ -273,6 +279,12 @@ class CTestNetParams : public CChainParams {
         base58Prefixes[EXT_PUBLIC_KEY] = {0x04, 0x35, 0x87, 0xCF};
         base58Prefixes[EXT_SECRET_KEY] = {0x04, 0x35, 0x83, 0x94};
 
+        base58EncodedPrefixes[PUBKEY_ADDRESS] = {"m","n"};
```

Comment:
Super nit: Personal preference as I find it congruent with the English language and it's slightly easier on the eyes. Same for all these other newly added constants.

```diff
- base58EncodedPrefixes[PUBKEY_ADDRESS] = {"m","n"};
+ base58EncodedPrefixes[PUBKEY_ADDRESS] = {"m", "n"};
```

Reply:
Yes, please format the modified code according to the specified formatter